css/style.css

```diff
@@ -1039,22 +1041,1370 @@ li { list-style-type: none; }
+  }
+  
+  .section__btn-pointer {
+    display: flex;
+    width: 360px;
+    height: 14px;
+    padding: 0;
+    margin: 0 auto;
+    box-shadow: none;
+    background: url(https://i.ibb.co/GFX8Xs7/Vector-1.png) no-repeat;
+    background-size: 7%;
+    background-position: center center;
+    outline: none;
+  }
+
+  /* -------2-------- */
+
+  .sections__dashboard {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 375px;
+    height: 100%;
+    align-content: flex-end;
+    margin: 0;
+    background: url(https://i.ibb.co/JkTYWY9/2013-tesla-model-s-cockpit-i-pad.png) no-repeat;
+    background-size: 137%;
+    background-position: top left -58px;
+  }
+  
+  .dashboard__header {
+    display: none;
+  }
+  
+  .dashboard-right {
+    display: flex;
+    flex-flow: row wrap;
+    max-width: 375px;
+    max-height: 487.21px;
+    align-items: center;
+    justify-content: center;
+    font-size: 40px;
+    color: white;
+  }
+  
+  .dashboard-right__sidebar {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    justify-content: flex-start;
+  }
+
+  .sidebar {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    justify-content: flex-start;
+    width: 375px;
+    max-height: 300px;
+    color: #fff;
+    background: #f00;
+    background-size: 100%;
+    margin: 325px 0 0;
+  }
+  
+  .sidebar__symbol {
+    display: none;
+  }
+
+  .sidebar__title {
+    display: flex;
+    justify-content: center;
+    width: 316px;
+    height: 37px;
+    color: #fff;
+    margin: 18px 0 9px 27px;
+  }
+
+  .sidebar__item {
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    width: 316px;
+    height: 206px;
+    padding: 0;
+    margin: 0 0 0 27px;
+  }
+
+  /* ---ссылка тест и кнопка заказать--- */
+
+  .info {
+    display: none;
+  }
+
+  .sidebar__info {
+    display: none;
+  }
+
+  /* -------3-------- */
+
+  .frame {
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: flex-start;
+    align-content: center;
+    max-width: 375px;
+    height: 100%;
+    margin: 0 auto;
+  }
+    
+  .frame__sidebar {
+    display: flex;
+    order: 2;
+    flex-flow: column wrap;
+    justify-content: flex-start;
+    align-content: center;
+    max-width: 375px;
+    max-height: 304px;
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    color: #fff;
+    background: #f00;
+    background-size: 100%;
+    margin: 0;
+  }
+  
+  .frame__title {
+    display: flex;
+    width: 316px;
+    height: 74px;
+    margin: 18px 0 11px 0;
+  }
+  
+  .frame__item {
+    display: flex;
+    width: 316px;
+    height: 199px;
+    margin: 0;
+  }
+  
+  /* Оценка защиты */
+  
+  .frame__title-rating {
+    display: none;
+  }
+  
+  .frame__result {
+    display: none;
+  }
+  
+  /* ссылка тест и кнопка заказать */
+
+  .frame__info {
+    display: none;
+  }
+
+    /* правая часть секции, каркас */
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)